static/app/components/feedback/feedbackSetupPanel.tsx

```diff
@@ -20,6 +21,7 @@ export default function FeedbackSetupPanel() {
     trackAnalytics('feedback.index-setup-viewed', {
       organization,
     });
+    SidebarPanelStore.hidePanel('feedback-sidequest');
```

Comment:
what does this do?

Reply:
this is what hides the panel upon page refresh

Reply:
the sidebar being active/hidden is controlled by this`SidebarPanelStore`

Reply:
ah, yeah i see it.